### ISSUE TITLE
scoverage plugin does not work on Scala 2.12.15

### ISSUE THREAD

**Issue by xkrogen** [scoverage/gradle-scoverage#174]
When using a project with `org.scoverage:gradle-scoverage:7.0.0` in combination with Scala 2.12.15, the build fails, because the plugin is trying to pull version 1.4.8 of `scalac-scoverage-plugin`, but `scoverage` only publishes Scala 2.12.15-compatible builds starting from version 1.4.9 ([refer to the version list on Maven Central](https://search.maven.org/artifact/org.scoverage/scalac-scoverage-plugin_2.12.15)).

I'm able to work around this for now by forcing the version up to 1.4.10 in my `build.gradle` as shown below, but obviously this is not ideal:
```groovy
dependencies {
  scoverage "org.scoverage:scalac-scoverage-plugin_2.12.15:1.4.10"
}
```

I assume this is a simple version bump somewhere inside of the plugin?

**Comment by mwos-sl** [comment]
Same story, and because I apply plugin like this:
```
plugins {
  id "org.scoverage" version "7.0.0" 
}
```

Fix in my case was to:
```
  scoverage {
    scoverageVersion = "1.4.10"
  }
```

**Comment by xkrogen** [comment]
Thanks @mwos-sl , that's a much better workaround!

**Comment by maiflai** [comment]
Released 7.0.1 with a default dependency of 1.4.11

**Comment by xkrogen** [comment]
Thanks @maiflai !
